### ISSUE TITLE
code exmaples to readme

### ISSUE THREAD

**Issue by antonkulaga** [drdozer/scalatags-ext#2]
It would be nice to see some code examples with new features

**Comment by drdozer** [comment]
Good idea @antonkulaga -- I'll see about extracting some examples when I've finished using some of this API in an actual project ;) Anything in particular you'd like to see used?
